portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includePage.vm

```diff
@@ -29,10 +29,13 @@
     #set( $numberTools = $pageColumn0Tools.size() )
 #end
 <div class="portal-main-container container-fluid mt-2">
-    <main id="$pageWrapperClass" class="portal-main-content d-flex flex-column #if( $numberTools > 1 || $homePage)Mrphs-multipleTools #end" role="main">
+    <main id="$pageWrapperClass" class="portal-main-content #if( $numberTools > 1 || $homePage)Mrphs-multipleTools #end" role="main">
         <h2 class="skip visually-hidden" tabindex="-1" id="tocontent">${rloader.sit_contentshead}</h2>
         #parse("/vm/morpheus/includeSiteHierarchy.vm")
         #parse("/vm/morpheus/snippets/siteStatus-snippet.vm")
+        #if ($pageTwoColumn)
+        <div class="d-flex">
+        #end
         <div id="$pageColumnLayout" class="${columnClasses}">
```

Comment:
I think we can do this in a single same div by using the ternary operator, make sure you test it whether this will work or not.
```suggestion
<div id="$pageColumnLayout" class="${pageTwoColumn ? 'd-flex ' : ''}${columnClasses}">
```